src/Tickets/Commerce/Gateways/Stripe/Settings.php

```diff
@@ -331,8 +331,29 @@ public function get_connection_settings_html() {
 			'merchant_status' => tribe( Merchant::class )->get_connection_status(),
 			'signup'          => tribe( Signup::class ),
 			'merchant'        => tribe( Merchant::class ),
+			'fee_is_applied'  => $this->should_apply_stripe_fee( true ),
 		];
 
 		return $admin_views->template( 'settings/tickets-commerce/stripe/main', $context, false );
 	}
+
+	/**
+	 * Should the processor fee be applied to this site's transactions?
+	 *
+	 * @since TBD
+	 *
+	 * @param bool $revalidate whether to submit a new validation API request
+	 *
+	 * @return bool
+	 */
+	public function should_apply_stripe_fee( $revalidate = false ) {
+
+		if ( class_exists( 'Tribe__Tickets_Plus__PUE' ) ) {
+			if ( tribe( \Tribe__Tickets_Plus__PUE::class )->is_current_license_valid( $revalidate ) ) {
+				return false;
+			}
+		}
```

Comment:
We should try to avoid having ET+ code on the free version. I feel the way to go here would be having a filter and move this method to ET+, hooking here.

Reply:
The problem with setting a filter here is that any user would be able to hook into it from their theme and return false, and then remove the fees without having a license.

Reply:
I agree with @moraleida in this case it's ok to have it not be a filter. But lets invert the method here. So it's not a nestted conditional.

Reply:
If they do that they deserve not to pay fees, lol! Your point is valid.